docs/package.json

```diff
@@ -49,15 +49,15 @@
   },
   "type": "module",
   "dependencies": {
-    "@web3-onboard/arcana-auth": "^2.0.0-alpha.1",
+    "@web3-onboard/arcana-auth": "^2.0.0",
     "@web3-onboard/blocto": "^2.0.0",
-    "@web3-onboard/cede-store": "^2.1.0-alpha.1",
-    "@web3-onboard/coinbase": "^2.2.5-alpha.1",
-    "@web3-onboard/core": "^2.21.0-alpha.1",
+    "@web3-onboard/cede-store": "^2.1.0",
+    "@web3-onboard/coinbase": "^2.2.5",
+    "@web3-onboard/core": "^2.21.0",
     "@web3-onboard/dcent": "^2.2.7",
     "@web3-onboard/enkrypt": "^2.0.4",
     "@web3-onboard/fortmatic": "^2.0.19",
-    "@web3-onboard/frame": "^2.0.2-alpha.1",
+    "@web3-onboard/frame": "^2.0.2",
     "@web3-onboard/frontier": "^2.0.4",
     "@web3-onboard/gas": "^2.1.8",
     "@web3-onboard/gnosis": "^2.1.10",
```

Comment:
I think the new gnosis deps will need to be added to the docs and react demo to work

Reply:
would that be in the docs PR or the main pr here?